core/controllers/acl_decorators_test.py

```diff
@@ -2164,16 +2164,13 @@ def test_assigning_voice_artist_for_unsupported_entity_type_raise_400(self):
                 'Unsupported entity_type: topic')
         self.logout()
 
-    def test_voiceover_admin_cannot_assign_voice_artist_in_private_exp(self):
+    def test_voiceover_admin_can_manage_voice_artist_in_private_exp(self):
```

Comment:
Just to check though ... will they be able to get into the editor page for a private exp? They shouldn't be able to, right? (But if they have this power as a different role, then it should be fine.)

(It may be worth clarifying this, otherwise it seems weird.)

Reply:
Done added a comment. They won't be able to access the editor.